diff --git a/qtbase/configure.pri b/qtbase/configure.pri
index 81c84b2..06ada45 100644
--- a/qtbase/configure.pri
+++ b/qtbase/configure.pri
@@ -383,8 +383,8 @@ defineTest(qtConfTest_detectPkgConfig) {
                 return(false)
             }
 
-            pkgConfigSysrootDir = $$sysroot
-            qtConfAddNote("PKG_CONFIG_SYSROOT_DIR automatically set to $$pkgConfigSysrootDir")
+            #pkgConfigSysrootDir = $$sysroot
+            qtConfAddNote("WARNING :: Qt upstream would've set PKG_CONFIG_SYSROOT_DIR automatically set to $$pkgConfigSysrootDir, we do not!")
         }
         $${1}.pkgConfigLibdir = $$pkgConfigLibdir
         export($${1}.pkgConfigLibdir)
