diff --git a/spyder/config/main.py b/spyder/config/main.py
index f5325bf..0f14fd5 100644
--- a/spyder/config/main.py
+++ b/spyder/config/main.py
@@ -89,7 +89,7 @@ DEFAULTS = [
               'custom_margin': 0,
               'use_custom_cursor_blinking': False,
               'show_internal_errors': True,
-              'check_updates_on_startup': True,
+              'check_updates_on_startup': False,
               'toolbars_visible': True,
               'cursor/width': 2,
               'completion/size': (300, 180),


