Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 2832)
+++ CMakeLists.txt	(working copy)
@@ -424,6 +424,10 @@
 ADD_LIBRARY(${GEOTIFF_LIBRARY_TARGET}
     ${GEOTIFF_LIB_SOURCES} ${GEOTIFF_LIB_CSV_SOURCES} ${XTIFF_SOURCES})
 
+IF(WIN32 AND MSVC)
+    SET_TARGET_PROPERTIES(${GEOTIFF_LIBRARY_TARGET} PROPERTIES IMPORT_SUFFIX "_i.lib")
+ENDIF(WIN32 AND MSVC)
+
 # Unix, linux:
 IF(UNIX)
    SET_TARGET_PROPERTIES(

